frontends/concrete-python/concrete/fhe/compilation/configuration.py

```diff
@@ -616,6 +616,115 @@ def _promotions(
         return required_x_bit_width, required_y_bit_width
 
 
+class MultivariateStrategy(str, Enum):
```

Comment:
Maybe it's the right time (or just after this already big PR) to split that file. There's a lot of strategies now.

Reply:
I totally agree, and min/max is on it's way with `MinMaxStrategy` :upside_down_face: